lua/wire/client/wire_expression2_editor.lua

```diff
@@ -1837,7 +1837,7 @@ function Editor:Close()
 	timer.Stop("e2autosave")
 	self:AutoSave()
 
-	self:Validate()
+	if self.E2 then self:Validate() end
```

Comment:
If you have that style elsewhere, better change that as well.
In object-oriented programming (and that's what derma is), you override methods in these cases instead of littering the code with ifs